src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -139,7 +139,8 @@ BLOCK_STRUCTURES_SETTINGS:
     TASK_MAX_RETRIES: 5
 BRANCH_IO_KEY: ''
 BUGS_EMAIL: odl-devops@mit.edu  # MODIFIED
-EMAIL_USE_DEFAULT_FROM_FOR_BULK: true
+EMAIL_USE_DEFAULT_FROM_FOR_BULK: true # Can be removed. It's renamed https://github.com/openedx/edx-platform/pull/29001
```

Comment:
If it doesn't do anything (since 2021), let's remove it -- can be in this PR or another one.

Reply:
I will create a separate PR for that. It's good to have a separate one in case this pull request gets reverted.